Syllabus/syllabus.Rmd

```diff
@@ -1,8 +1,8 @@
 ---
 title: "Syllabus"
 output:
-  word_document: default
   html_document: default
+  word_document: default
```

Comment:
Gotta be careful about this whole part - I always forget that the knit button only renders the first one.